routes/truckRoutes.js

```diff
@@ -1,3 +1,118 @@
-// connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
+// connect to mongodb in this module as this is where you'll be making create/read/delete calls to your database
 // use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
 // remember this is a Node module
+var express = require('express');
+var app = express.router();
+
+// mongoose is just another node module
+var mongoose = require('mongoose');
+var db = mongoose.connect('mongodb://localhost/trucksAPI');
+var bodyParser = require('body-parser');
+
+var Truck = require('./models/truckModel');
+var idManager = require('./idManager');
+
+idManager.setIds(trucks.getTrucks());
+var serveStatic = express.static('public');
+app.use(serveStatic);
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+//A GET route for '/trucks' that sends to the response the list of all food-truck objects in JSON format.
+app.get('/trucks', function (request, response){

```

Comment:
This isn't how you use the router instance - this is how you would use an application instance.